src/breadcrumb.component.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import { Button } from './button.component'
+import { getColor } from './provider.component'
+
+import useWindowSize from '@rehooks/window-size'
+
+import { Anchor } from 'grommet'
+
+
+export function Breadcrumb( props ) {
+  
+  const windowSize = useWindowSize()
+  const showFull = ( windowSize.innerWidth > 900 ) ? true : false;
+
+  return (
+      <div style={{minHeight:'2rem'}}>
+
+        { (!showFull) ?
+          <StyledButton
+          label={"Übersicht Prozentrechnung"}
+          className={props.className}
+          a11yTitle={props.title}
+          plain
+          iconName={'arrow-circle-left'}
+          
+          backgroundColor={getColor('lighterblue')}
+          activeBackgroundColor={getColor('brand')}
+        />
+          :
+          <BreadcrumbList>
+            <Anchor href="#">Mathematik </Anchor>
+            &nbsp;>&nbsp;<Anchor href="#">Terme und Gleichungen</Anchor>
```

Comment:
You can also do `{' '}` if you prefer that

Reply:
ah yeah in this case `{' '}>&nbsp;` would make sense